com.unity.cinemachine/Editor/Samples/SampleDependencyImporter.cs

```diff
@@ -170,6 +182,64 @@ static bool PromptUserImportDependencyConfirmation(List<string> dependencies)
                     "Import samples without their dependencies");
             }
         }
+
+#if CINEMACHINE_URP || CINEMACHINE_HDRP
+        static readonly string[] k_MaterialFolders = { "Materials", "Cameron/Model" };
+        static void ConvertMaterials(IEnumerable<string> folders)
+        {
+            foreach (var folder in folders)
+            {
+                foreach (var materialFolder in k_MaterialFolders)
+                {
+                    var materialDir = new DirectoryInfo(folder + "/" + materialFolder);
+                    if (!materialDir.Exists)
+                        continue;
+                    var materialInfos = materialDir.GetFiles("*.mat");
+                    foreach (var matInfo in materialInfos)
+                    {
+                        var localPath = matInfo.FullName[Application.dataPath.Length..];
+                        var material = AssetDatabase.LoadAssetAtPath<Material>("Assets/" + localPath);
+
+                     MaterialUpgrader.Upgrade(material, 
+    #if CINEMACHINE_URP
+                         new UnityEditor.Rendering.Universal.StandardUpgrader(material.shader.name), 
+    #elif CINEMACHINE_HDRP
+                         MaterialUpgradeHelper.GetHDRPMaterialUpgraders(),
+    #endif
+                         MaterialUpgrader.UpgradeFlags.None);
+                    }
+                }
+            }
+        }
+
+        static void FixPrefabs(IEnumerable<string> folders)
+        {
+    #if CINEMACHINE_HDRP
+            foreach (var folder in folders)
+            {
+                var hdrpPath = folder + "/~HDRP/";
```

Comment:
Somehow the ~HDRP folder ends up in the project when I import the samples.  It shouldn't be there, I think.  If the ~ strategy doesn't work, maybe we can think of something else.  
 
![image](https://user-images.githubusercontent.com/6424658/225380721-d4c9aabf-7f0d-4c5e-8986-e265b2e1dcdb.png)

Reply:
I think I can stop it from being imported. I'll check.

Reply:
I cannot stop it from being imported, but I can clean it up when it is not needed anymore (e.g. no HDRP or we have swapped out the prefabs). User won't see it appear.